src/constants/env.ts

```diff
@@ -0,0 +1,3 @@
+import { unwrapOrThrow } from '@utils/index'
```

Comment:
I put stuff these on https://github.com/fuseio/subscription-service/blob/master/config/custom-environment-variables.json

Reply:
also need this accordingly